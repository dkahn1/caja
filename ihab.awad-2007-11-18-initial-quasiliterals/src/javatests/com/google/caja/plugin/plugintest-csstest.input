=== test:///position.css
@template('position');
@param('left');
@param('top');

left: $(left)px; top: $(top)px


=== test:///csstest.js
function main() {
  getElementById('hello-base').setStyle(position(20, 20));
}
