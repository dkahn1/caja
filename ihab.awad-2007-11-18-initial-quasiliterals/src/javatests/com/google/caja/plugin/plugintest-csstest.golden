{
  var MY_TEST_PLUGIN_PRIVATE = {
    'nsPrefix': 'pre',
    'pathPrefix': '/plugin1',
    'name': 'MY_TEST_PLUGIN'
  };
  var MY_TEST_PLUGIN = {
    'position': function position(left, top) {
      var ___out___ = [];
      ___out___.push('left: ', plugin_cssNumber___(left), 'px;\ntop: ', plugin_cssNumber___(top), 'px');
      return plugin_blessCss___(___out___.join(''));
    }
  };
  MY_TEST_PLUGIN_PRIVATE.plugin = MY_TEST_PLUGIN;
  plugin_initialize___(MY_TEST_PLUGIN_PRIVATE);
  (function () {
      MY_TEST_PLUGIN.main = function main() {
        (MY_TEST_PLUGIN.getElementById('hello-base')).setStyle(MY_TEST_PLUGIN.position(20, 20));
      };
    }).call(MY_TEST_PLUGIN);
}
